modules/blackhole-monitor/lambda.tf

```diff
@@ -0,0 +1,27 @@
+data "archive_file" "source" {
```

Comment:
Consider using [this module](https://github.com/ministryofjustice/hmpps-ems-platform-terraform-modules/tree/main/modules/standard-lambda). It should reduce the amount of duplication we have.

Reply:
What's the best approach for this? Do I use standard-lambda as a source in this repo/module or from https://github.com/ministryofjustice/hmpps-ems-platform like you have with https://github.com/ministryofjustice/hmpps-ems-platform/blob/main/accounts/hmpps-iam/inactive-account-monitor/lambda.tf#L31

Reply:
Yeah, I think thats the right way to go. And we should be building out that module as we go along.

What do you think?